README.md

```diff
@@ -193,6 +193,20 @@ This codebase follows [single-direction margin declarations](https://csswizardry
 
 Margins are generally preferred to padding for whitespace (due to [margin collapsing](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Box_Model/Mastering_margin_collapsing)), but the latter can be used when relevant.
 
+Accessibility
+------------
+
+### Screen Reader Observations
+
+#### Voiceover (Mac)**
+
+Download Link Icon
+
+We have noticed that Voiceover does not read aria-labels in anchor tags `<a href="" arial-label="Voiceover"></a>` in Safari when you load or reload the page. The only time it seems to read it out is if you tab to another piece of software or browser and then tab back to Safari.
+
+To overcome this we have removed the aria-label from the anchor tag and put the label description into the image alt tag instead. This means the description gets read out consistently.
+
```

Comment:
The overall structure above is good: the first paragraph states the problem, the second paragraph describes a solution.

First paragraph: the code example could be slightly tightened up, something like `<a href="..." arial-label="Voiceover ignores this"></a>`.

Second paragraph: rather than saying what we've done in a specific case, better to phrase in terms of how to address the problem for the general case.

Reply:
I think at this point we will need to add visuallyhidden so we can add a span for links that do not have an image. What do you think?

Reply:
I think keep it focused, so para 2 could be something like:

"When the link contains an image but no text (and so a description is essential), one work around is removing the anchor element's `aria-label`, and moving its descriptive text to the `alt` attribute."